src/coreclr/src/pal/src/misc/environ.cpp

```diff
@@ -917,44 +917,51 @@ Return Value
     or nullptr otherwise.
 
 --*/
+static char* match_prefix(const char* name, char** list)
```

Comment:
Can you please move this function above the comment that belongs to the EnvironGetenv?